src/sass/components/_button.scss

```diff
@@ -1,36 +1,52 @@
-.btn-main {
-  background-color: #2a2a2a;
-  color: #ffffff;
-  font-size: 16px;
-  line-height: 24px;
-  font-weight: 600;
-  padding: 10px 30px;
+.product-box {
 
-  &:not(.no-hover):hover {
-    color: #ffffff !important;
-    background-color: $primary;
-    text-decoration: none;
-  }
-}
+  .btn-main {
+    background-color: #2a2a2a;
+    color: #ffffff;
+    font-size: 16px;
+    line-height: 24px;
+    font-weight: 600;
+    padding: 10px 30px;
 
-.btn-main-small {
-  @extend .btn-main;
-  font-size: 13px;
-  line-height: 24px;
-  padding: 5px 10px;
-  font-weight: 400;
-}
+    &:not(.no-hover) {
+      opacity: 0;
+      text-decoration: none;
+    }
+  }
 
-.btn-outline {
-  display: inline-block;
-  padding: 5px 10px;
-  border: 1px solid #2a2a2a;
-  color: #2a2a2a;
-  font-size: 13px;
-  line-height: 22px;
+  .btn-main-small {
+    @extend .btn-main;
+    font-size: 13px;
+    line-height: 24px;
+    padding: 5px 10px;
+    font-weight: 400;
+  }
 
   &:hover {
-    background-color: #2a2a2a;
-    color: #ffffff;
+    .btn-main {
+      color: #ffffff !important;
+      background-color: $primary;
+      text-decoration: none;
+      opacity: 1;
+      transition: 0.45s;
+
+      &:not(.no-hover) {
+        background-color: #2a2a2a;
```

Comment:
please define a variable for that color and use it here and in other places below